packages/nextjs/src/utils.ts

```diff
@@ -0,0 +1,16 @@
+import { NhostSession } from '@nhost/client'
+
+export const refresh = async (
+  nhostUrl: string,
+  refreshToken: string
+): Promise<NhostSession | null> => {
+  const result = await fetch(`${nhostUrl}/v1/auth/token`, {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json'
+    },
+    body: JSON.stringify({ refreshToken })
+  })
+  if (result.status >= 400) return null
```

Comment:
Can we use `result.ok` here instead? Just asking because I don't really know what status codes should result in a nullish result.